### PR TITLE
Get user interactions memos

### PR DESCRIPTION
**Pull request by arfenis**

### CODE REVIEW

File: library/Models/StormsInteractions.php [hunk 1]
```diff
@@ -59,4 +68,45 @@ public function removeInteraction(): void
...
+
+    /**
+     * Return the storms that the user interact with by the interaction type
+     *
+     * @param integer $userId
+     * @param string $interactionType
+     * @throws Exception
+     * @return array
+     */
+    public static function getStormsInteractionByUser(int $userId, string $interactionType): array
```
Comment (kaioken) [inline library/Models/StormsInteractions.php]: @arfenis send user instead of Id?

File: library/Models/StormsInteractions.php [hunk 2]
```diff
@@ -59,4 +68,45 @@ public function removeInteraction(): void
...
+    {
+        $storms = [];
+
+        switch ($interactionType) {
```
Comment (kaioken) [inline library/Models/StormsInteractions.php]: @arfenis has this specify on a function to check?

File: api/controllers/FeedsController.php [hunk 3]
```diff
@@ -89,20 +89,19 @@ public function addToList(string $id): Response
...
         $feedType = $this->request->getQuery('type', 'string', '');
 
-        $this->additionalSearchFields = [
-            ['is_deleted', ':', 0],
-            ['users_id', ':', $id]
-        ];
+        if (!empty($feedType)) {
+            $results = StormsInteractions::getStormsInteractionByUser($id, $feedType);
+    
+            return $this->response(!$results ? $results : $this->processOutput($results));
```
Comment (kaioken) [inline api/controllers/FeedsController.php]: @arfenis umm didnt understand this results , if it doesnt exist return null?

File: library/Models/StormsInteractions.php (hunk 1 above)
Comment (arfenis) [reply library/Models/StormsInteractions.php]: That imply get the user by the $id sent in the route.

File: api/controllers/FeedsController.php (hunk 3 above)
Comment (arfenis) [reply api/controllers/FeedsController.php]: The function return an empty array if no storms is found.

**Pull request merged**